CSharpBasics/CSharpBasics3/Program.cs

```diff
@@ -0,0 +1,136 @@
+            {
+                Console.WriteLine("frutti");
+            }
+            else
+            {
+                Console.WriteLine("You number isn't divisible by 2 or 5 without a remainder");
+            }
+        }
+        catch (Exception)
+        {
+            Console.WriteLine("You entered not number");
+        }
+
+    }
+
+    public void Task_1_2()
+    {
+        Console.WriteLine("Enter any number");
+        bool parseNumber = int.TryParse(Console.ReadLine(), out var number);
+        switch (parseNumber)
+        {
+            case true when (number % 5 == 0 && number % 2 == 0):
+                Console.WriteLine("tutti-frutti");
+                break;
+            case true when (number % 2 == 0):
+                Console.WriteLine("tutti");
+                break;
+            case true when (number % 5 == 0):
+                Console.WriteLine("frutti");
+                break;
+            case true when (number % 5 != 0 && number % 2 != 0):
+                Console.WriteLine("You number isn't divisible by 2 or 5 without a remainder");
+                break;
+            default:
+                Console.WriteLine("You entered not number");
+                break;
+        }
+
+    }
+    public void Task_2()
+    {
+        while (true)
+        {
+            try
+            {
+                Console.WriteLine("Enter the start number:");
+                int startNumber = int.Parse(Console.ReadLine());
+
+                Console.WriteLine("Enter the end number:");
+                int endNumber = int.Parse(Console.ReadLine());
+
+                for (int i = startNumber; i <= endNumber; i++)
+                {
+                    if (i % 5 == 0 && i % 2 == 0)
+                    {
+                        Console.WriteLine("tutti-frutti");
+                    }
+                    else if (i % 2 == 0)
+                    {
+                        Console.WriteLine("tutti");
+                    }
+                    else if (i % 5 == 0)
+                    {
+                        Console.WriteLine("frutti");
+                    }
+                    else
+                    {
+                        Console.WriteLine($"Number {i} can't be divided by 2 or 5 without a remainder");
+                    }
+                }
+            }
+            catch (Exception)
+            {
+                Console.WriteLine("You entered not number");
+                break;
+            }
+        }
+    }
+    public void Task_3()
+    {
+        Console.WriteLine("Enter the first number:");
+        int firstNumber = int.Parse(Console.ReadLine());
+
+        Console.WriteLine("Enter the second number:");
+        int lastNumber = int.Parse(Console.ReadLine());
+
+        int minNumber = Math.Min(firstNumber, lastNumber);
+        int maxNumber = Math.Max(firstNumber, lastNumber);
+
+        //    int minNumber = (firstNumber < lastNumber) ? firstNumber : lastNumber;
```

Comment:
do not forget to remove unused comments :) in future it will make your code much cleaner :)